godtools/Managers/DownloadResources/DownloadedResourceManager.swift

```diff
@@ -29,18 +29,18 @@ class DownloadedResourceManager: GTDataManager {
         serializer.registerResource(AttachmentResource.self)
     }
     
-    func loadFromDisk() -> [DownloadedResource] {
+    func loadFromDisk() -> List<DownloadedResource> {
         resources = findAllEntities(DownloadedResource.self)
         return resources
     }
     
-    func loadFromRemote() -> Promise<[DownloadedResource]> {
+    func loadFromRemote() -> Promise<List<DownloadedResource>> {
         showNetworkingIndicator()
         
-        let params = ["include": "translations,pages,attachments"]
+        let params = ["include": "translations,ages,attachments"]
```

Comment:
This is a good catch. We actually don't need to retrieve pages anymore, so this can be removed. I think it makes sense to do in #70 though.